platform/ios/src/MGLMapView.mm

```diff
@@ -1223,6 +1258,12 @@ - (void)handlePanGesture:(UIPanGestureRecognizer *)pan
         {
             CGPoint offset = CGPointMake(velocity.x * self.decelerationRate / 4, velocity.y * self.decelerationRate / 4);
             _mbglMap->moveBy({ offset.x, offset.y }, MGLDurationInSeconds(self.decelerationRate));
+            __weak MGLMapView *weakSelf = self;
```

Comment:
I tried that first but then it'll fade out while you zoom or pan if you don't move consistently. :/